lib/util.ml

```diff
@@ -1004,14 +1004,18 @@ let geneweb_link conf href s =
 let wprint_geneweb_link conf href s =
   Wserver.printf "%s" (geneweb_link conf href s)
 
-let reference conf base p s =
+let reference_flags with_id conf base p s =
   let iper = get_iper p in
   if conf.cancel_links || is_hidden p then s
   else
     String.concat ""
-      ["<a href=\""; commd conf; acces conf base p; "\" id=\"i";
-       string_of_iper iper; "\">"; s; "</a>"]
+      ["<a href=\""; commd conf; acces conf base p;
+       if with_id then string_of_iper iper else "";
```

Comment:
@lledieu 

I think this should be `"\" id=\"i" ^ string_of_iper iper else "";`

Reply:
yes

Reply:
Mon dieu, c'est vieux ! Faut que je retrouve pourquoi j'ai fait ce changement...

Reply:
Oui, bon anniversaire en retard à la PR du coup :birthday:

Reply:
Il me semble avoir supprimé l'attribut id car on avait dans certains cas des doublons (même id pour plusieurs objets dans la même page) et que j'avais juré que l'attribut ne servait à rien... Je continue à creuser un peu.

Reply:
Dans ma version locale qui fonctionne pour moi depuis 1 an j'ai : 
`let reference_flags with_id conf base p s =
  let iper = get_key_index p in
  if conf.cancel_links || is_hidden p then s
  else
    String.concat ""
      ["<a href=\""; commd conf; acces conf base p;
       if with_id then "\" id=\"i" else "";
       if with_id then string_of_int (Adef.int_of_iper iper) else "";
       "\">"; s; "</a>"]`

Reply:
À remettre donc, ok.

Reply:
On a reperdu l'id lors du rebase je pense.

Reply:
Décidément ! J'espère que cette fois-ci c'est la bonne...

Reply:
On  dirait bien.